src/routes/balances/balances.controller.spec.ts

```diff
@@ -61,7 +63,7 @@ describe('Balances Controller (Unit)', () => {
     await app.close();
   });
 
-  describe('GET /balances', () => {
+  describe('Maps ERC20 token correctly', () => {
```

Comment:
Maybe this title should remain the same (`GET /balances`)? 🤔

Reply:
> Maybe this title should remain the same (`GET /balances`)? 🤔

I wanted to draw some comparison with the new test which also tests `GET /balances` in a way. Do you have other suggestions?

Reply:
Maybe I'm not getting it, but you are changing the `describe` which contains two tests. The first one could have `it('Maps ERC20 token correctly', ...)` and the second one `it('Maps native token correctly', ...)`, but the `describe` block should remain the same, right? 🙂

Reply:
[f9fb1a6](https://github.com/safe-global/safe-client-gateway-nest/pull/330/commits/f9fb1a68f747ee51909a330d04e89a7bf8968cda)